index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="author" content="Brandon and Tiago">
+  <link rel="icon" type="image/png" href="images/intuit-icon.png">
+  <link rel="apple-touch-icon" type="image/png" href="images/intuit-icon.png"/><!--Apple Compatibility1-->
+  <title>Mint Sign Up Clone</title>
+  <!--     <link href="styles/reset.css" rel="stylesheet" type="text/css">   use to erase preset styles as debug tool -->
```

Comment:
Please remove all comments from your `index.html` file. This is required by Microverse to approve the project, and it makes your code look clean! ✨